### DISCUSSION TITLE
Notes speeding up with bpm changes while mapping in osu!mania.

### DISCUSSION THREAD

**Discussion by xdTAGCLAN** [ppy/osu#16370] [Q&A] [answered]
This isn't really and issue but I couldn't find a fix for it on google. So basically I'm making a beatmap with a lot of bpm changes and the note speed changes with the bpm changes. How do I stop this from happening?

**Comment by peppy** [comment, marked as answer]
That is how osu!mania works. If you don't want this, you would need to counter the speed changes with "scroll speed" adjustments. But keep in mind the speed changes allow the user to know that the bpm is changing. Bypassing this isn't recommended.

**Reply by Walavouchey** [reply]
FWIW, the [mania ranking criteria](https://osu.ppy.sh/wiki/en/Ranking_Criteria/osu%21mania) recommend normalisation of SV apart from SV effects.

Under guidelines:

> Timing changes should have a unified scroll speed by adjusting slider velocity. 

and

> Slider velocity changes should follow the general intensity of the song.

Based on that, I presume it is undesirable to let BPM (instead of intensity) dictate scroll speed, especially on variable-BPM maps.

**Reply by peppy** [reply]
trust the ranking criteria to go against all common sense. this is very subjective guidance that should not be in such a document.
